src/components/addButton.tsx

```diff
@@ -0,0 +1,16 @@
+import addbutton from "../assets/icons/addbutton.svg";
+
+interface AddButtonProps {
+  openModal: () => void;
```

Comment:
Suggestion: change the name of the prop to onClick. I know that in this case you are using this button to open a modal, but in the future, it might be used for another thing. Also, the developers that may want to use this component might think that this prop could be related to a modal that is implemented in this button, without the need for them to implement the modal itself, which could lead into a confusion.

Reply:
i changed the name from openModal to onClick as you said